goodtablesio/models/job.py

```diff
@@ -110,15 +113,53 @@ def get_ids():
     return [j.id for j in job_ids]
 
 
-def get_all():
-    """Get all jobs in the database as dict.
+def find(filters=None, limit=10, offset=0):
```

Comment:
WDYT about a thought I have written here - https://github.com/frictionlessdata/goodtables.io/pull/88 - no wrapping SA ORM interface into helpers? For example this `max_jobs_limit` is much more `view` parameter related for example to concrete data representation on some page.